controller/space_iterations_test.go

```diff
@@ -111,7 +110,7 @@ func (rest *TestSpaceIterationREST) TestSuccessCreateIteration() {
 		rootItr = &iteration.Iteration{
 			SpaceID:    newSpace.ID,
 			Name:       newSpace.Name,
-			UserActive: &userActive,
+			UserActive: false,
```

Comment:
Remove this line, since `UserActive` will automatically be `false`.

Reply:
Removed it here b92020c